modules/ROOT/nav.adoc

```diff
@@ -26,13 +26,14 @@
  ** xref:deploy-to-rtf-cli.adoc[Managing Mule Applications from the CLI]
  ** xref:deploy-maven-4.x.adoc[Deploy Applications Using Maven (Mule 4)]
  ** xref:deploy-maven-3.x.adoc[Deploy Applications Using Maven (Mule 3)]
- ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
- ** xref:use-log4j-appender.adoc[Use a Log4j Appender]
  ** xref:protect-app-properties.adoc[Protect App Property Values]
  ** xref:manage-secure-properties.adoc[Protect App Property Values with rtfctl]
  ** xref:app-versioning.adoc[Review Application Versioning]
  ** xref:app-to-app-requests.adoc[Send Application-to-Application Requests]
  ** xref:proxy-deploy-runtime-fabric.adoc[Deploy API Proxies]
+* xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
```

Comment:
Can we please add a link to this page in release notes to surface this doc change to customers:
"Review the resource allocation changes of anypoint monitoring agent which is used to monitor applications deployed on runtime fabric here<link>"